src/web/views/enter-code.njk

```diff
@@ -33,6 +33,6 @@
     {% endcall %}
 
     {# TODO DW HTBHF-1712 update link to send new confirmationCode  #}
-    <p class="govuk-body-m"><a href="#">{{ requestNewCode }}</a> {{ ifItDoesNotArrive }}</p>
+    <p class="govuk-body-m"><a href="#">{{ requestANewCode }}</a> {{ ifItDoesNotArrive }}</p>
```

Comment:
I think the translation key `requestANewCode` could have remained as `requestNewCode` then this update is not required. I personally don't think the key names need to reflect the text exactly.